SI/Task09.cpp

```diff
@@ -0,0 +1,47 @@
+#include <iostream>
+
+int Broqch(char* arr)
```

Comment:
Change the name of the function, const char* arr - you are not modifying the char[]